.github/actions/spelling/candidate.patterns

```diff
@@ -547,7 +547,8 @@ customObjectInstantitationMethod
 # base64 encoded content
 ([`'"])[-a-zA-Z=;:/0-9+]+=\g{-1}
 # base64 encoded content in xml/sgml
->[-a-zA-Z=;:/0-9+]+=</
+# Commented out because of false positives
```

Comment:
I think this false positive is only present in v0.0.21 and will go away in v0.0.22 (one of the many bugs that are fixed). But it doesn't matter.

Reply:
Ah my bad

Reply:
I should have called it out in the PR description.

In the [release notes](https://github.com/check-spelling/check-spelling/releases/tag/v0.0.22), it's listed as:
> 🐜 Bug fixes
> [🩹 Pattern suggestions shouldn't trip over active patterns](https://github.com/check-spelling/check-spelling/wiki/Feature:-Suggest-patterns#multiple-patterns-for-a-line)